e2e/cypress/integration/system_console/inactive_users_spec.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
```

Comment:
Can we add a pre-requisite to this test to ensure that default users are created in the system using the command:
./mattermost sampledata -w 4 -u 60 --deactivated-users 200

Reply:
Sure, I'll add that

Reply:
Now updated, PTAL, thanks